logic/account.js

```diff
@@ -177,25 +118,25 @@ function Account (db, schema, logger, cb) {
 			expression: '("balance")::bigint'
 		},
 		{
-			name: 'u_balance',
+			name: 'votes',
 			type: 'BigInt',
 			filter: {
 				required: true,
 				type: 'integer',
 				minimum: 0,
-				maximum: constants.totalAmount
+				maximum: constants.activeDelegates
 			},
 			conv: Number,
-			expression: '("u_balance")::bigint'
+			expression: '("votes")::int'
 		},
 		{
-			name: 'rate',
+			name: 'voters',
 			type: 'BigInt',
 			filter: {
 				type: 'integer'
 			},
 			conv: Number,
-			expression: '("rate")::bigint'
+			expression: '("voters")::int'
```

Comment:
`d.voters_cnt`, or should it contains array or voters addresses/pks?

Reply:
its a total count of whos voting, not array of address/pk